Directory.Packages.props

```diff
@@ -129,5 +129,6 @@
     -->
     <PackageVersion Include="Microsoft.IO.Redist" Version="6.0.1" />
     <PackageVersion Include="System.Formats.Asn1" Version="8.0.1" />
+    <PackageVersion Include="System.IO.Packaging" Version="6.0.1" />
```

Comment:
This seems surprisingly old, but if the build doesn't have a problem with it, I'm not going to overrule.

Reply:
Yea I'm surprised we're not using `8.0.1`. Here's the `dotnet nuget why` explanation:

```
  [net472]
  [net8.0]
   │  
   └─ Microsoft.AspNetCore.Razor.Test.Common.Tooling (v9.0.0-dev)
      └─ Microsoft.VisualStudio.Text.Implementation (v17.12.145-preview)
         └─ System.IO.Packaging (v6.0.1)
```